NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Em database explody

## Link to Ticket
Closes #14 

## Description of Proposed Changes

- Created directory structure and shell script for destroying and rebuilding the database and reseeding the new database.
- Added instructions to README for using the faker script
- There were several database changes to get Faker to work. These have been documented in the README.


## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout EM_DatabaseExplody
```

1. Start your virtual environment.
2. Run ```pip install django_seed```
3. If you have DB Browser open, close it.
3. Follow the directions in the README under "Reset the Database" for setting up & running the shell script
4. Open the database in DB Browser and verify that all of the tables are there and they have been populated with faker data.

## Impacted Areas in Application

Several of the models had changes to their field data types in order to make the faker data work.

## Mentions @username

@JacobAndrewSmith92 @obscure-shadow
